static/extensions/Codefoxy/cfupload.js

```diff
@@ -158,48 +242,102 @@
           apiurl = "https://file.io/?title=" + args.name;
           apiformname = "file";
           break;
-      }
-      return uploadFileToLink(args.data, args.name, apiurl, apiformname).then(
-        (response) => {
-          try {
-            a = JSON.parse(response); // Assuming response is JSON
-            return a;
-          } catch (error) {
-            console.error("Error parsing response as JSON:", error);
-            return response;
+        case "0x0.st":
+          apiurl = "https://0x0.st";
+          apiformname = "file";
+          break;
+        case "pastebin.com":
+          apiurl = "https://pastebin.com/api/api_post.php";
+          let fileData = args.data;
+
+          if (args.base64 === "true") {
+            if (!isBase64(args.data)) {
+              return "Base64 error: Invalid data";
+            }
+            try {
+              fileData = atob(args.data);
+            } catch (e) {
+              return "Base64 error: Invalid data";
+            }
           }
-        }
-      );
-    }
 
-    uploadFileToLink(args) {
```

Comment:
Idk if a block with this opcode exists, if so, you should not be removing it

Reply:
i changed to ``const uploadFileToLinkFunc = (){ }``